starknet_py/tests/e2e/utils.py

```diff
@@ -16,13 +16,28 @@
 AccountToBeDeployedDetails = Tuple[int, KeyPair, int, int]
 
 
-async def get_deploy_account_details(
-    *,
+def _new_address(
     class_hash: int,
+    calldata: List[int],
+):
+    salt = _get_random_salt()
+    return (
+        compute_address(
+            salt=salt,
+            class_hash=class_hash,
+            constructor_calldata=calldata,
+            deployer_address=0,
+        ),
+        salt,
+    )
+
+
+async def prepay_account(
+    *,
+    address: int,
```

Comment:
param `class_hash` needs to be changed to `address` and `argent_calldata` needs to be removed.